python/dgl/graphbolt/minibatch.py

```diff
@@ -474,6 +474,49 @@ def node_pairs_with_labels(self):
         else:
             return None
 
+    def to_pyg_data(self):
+        """Construct a PyG Data from `MiniBatch`. This function only supports
+        node classification task on a homogeneous graph and the number of
+        features cannot be more than one.
+        """
+        from torch_geometric.data import Data
+
+        if self.sampled_subgraphs is None:
+            edge_index = None
+        else:
+            col_nodes = []
+            row_nodes = []
+            for subgraph in self.sampled_subgraphs:
+                if subgraph is not None:
+                    sampled_csc = subgraph.sampled_csc
+                    indptr = sampled_csc.indptr
+                    indices = sampled_csc.indices
+                    expanded_indptr = expand_indptr(
+                        indptr, dtype=indices.dtype, output_size=len(indices)
+                    )
+                    col_nodes.append(expanded_indptr)
+                    row_nodes.append(indices)
+            col_nodes = torch.cat(col_nodes)
+            row_nodes = torch.cat(row_nodes)
+            edge_index = torch.unique(
+                torch.stack((col_nodes, row_nodes)), dim=1
+            )
+
+        if self.node_features is None:
+            node_features = None
+        else:
+            assert (
+                len(self.node_features) == 1
+            ), "`to_pyg_data` only supports single feature homogeneous graph."
```

Comment:
if graph is hetero while node_feature is None or len()==1, anything works well?

Reply:
If the node_feature is None or len()==1 it will be treated as a homogeneous graph.